pandas/tests/test_series.py

```diff
@@ -3008,6 +3008,27 @@ def test_round(self):
                           name='ts')
         assert_series_equal(result, expected)
         self.assertEqual(result.name, self.ts.name)
+ 
+        s = Series([1.123, 2.123, 3.123], index=lrange(3))               
+        outarr = np.empty(3)
+        result = s.round(out=outarr)
+        # outarr should be modified
+        self.assert_numpy_array_equal(result, outarr)
+        
+    def test_built_in_round(self):
+        if sys.version < LooseVersion('3.0'):
+            raise nose.SkipTest('build in round cannot be overriden prior to Python 3')
+        
+        s = Series([1.123, 2.123, 3.123], index=lrange(3))
+        result = round(s)
```

Comment:
futher there is already a ``test_round`` test, at the very least put these right after it